segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/extendeds3/ExtendedS3Storage.java

```diff
@@ -429,7 +429,7 @@ private Throwable translateException(String segmentName, Throwable e) {
             retVal = new ArrayIndexOutOfBoundsException(e.getMessage());
         }
 
-        if (e instanceof AccessDeniedException) {
+        if (e instanceof AccessControlException) {
```

Comment:
I'm not sure this is correct because of the following argument:

`S3JerseyClient` requests invoke this method to execute requests:

https://github.com/EMCECS/ecs-object-client-java/blob/master/src/main/java/com/emc/object/s3/jersey/S3JerseyClient.java#L719

`executeRequest` invokes this method `ErrorFilter.parseErrorResponse`:

https://github.com/EMCECS/ecs-object-client-java/blob/master/src/main/java/com/emc/object/s3/jersey/ErrorFilter.java#L104

which returns an `S3Exception` with the HTTP status code:

https://github.com/EMCECS/ecs-object-client-java/blob/master/src/main/java/com/emc/object/s3/jersey/ErrorFilter.java#L141

As such, I don't think this exception handling here is correct.